equipment_handler/inc/drivers/sci.h

```diff
@@ -1,168 +1,168 @@
-/** @file sci.h

```

Comment:
Not sure how this is a change. I didn't touch this file and they are identical anyways.

Reply:
It's because of moving it.